wasm/jit/jit_arm64.go

```diff
@@ -451,18 +476,214 @@ func (c *arm64Compiler) compileMemorySize() error {
 	return fmt.Errorf("TODO: unsupported on arm64")
 }
 
+// compileConstI32 implements compiler.compileConstI32 for the arm64 architecture.
 func (c *arm64Compiler) compileConstI32(o *wazeroir.OperationConstI32) error {
-	return fmt.Errorf("TODO: unsupported on arm64")
+	return c.emitIntConstant(true, uint64(o.Value))
 }
 
+// compileConstI64 implements compiler.compileConstI64 for the arm64 architecture.
 func (c *arm64Compiler) compileConstI64(o *wazeroir.OperationConstI64) error {
-	return fmt.Errorf("TODO: unsupported on arm64")
+	return c.emitIntConstant(false, o.Value)
 }
 
+// emitIntConstant adds instruction to load an integer constant.
+// is32bit is true if the target value is originally 32-bit const, false otherwise.
+// value holds the (zero-extended for 32-bit case) load target constant.
+func (c *arm64Compiler) emitIntConstant(is32bit bool, value uint64) error {
+	if value == 0 {
+		c.pushZeroValue()
+	} else {
+		// Take a register to load the value.
+		reg, err := c.allocateRegister(generalPurposeRegisterTypeInt)
+		if err != nil {
+			return err
+		}
+
+		// Then, load the const.
```

Comment:
this section down looks close to or the same for the float case. can this be extracted and reused if for no other reason re-using the comments around From.Offset?

Reply:
cool I found a lot of duplication which I was too enough to consolidate (in amd64).. arm64 code would be much less code in this way!

Reply:
https://github.com/tetratelabs/wazero/pull/212/commits/c3268d5957349f848ba654b83bf7c5a34a046b82